release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1099,6 +1101,8 @@ With this release, replacement control plane nodes are assigned to the correct i
 [id="ocp-4-13-storage-bug-fixes"]
 ==== Storage
 
+* With this update, the default credentials request for AWS has been modified to allow customer-managed key to be used for re-encryption in the KMS. Administrators who created credentials requests in manual mode with CCO must apply those changes manually by adding `kms:ReEncrypt*` permission to their key policy. Other administrators are not impacted by this change. (link:https://issues.redhat.com/browse/OCPBUGS-5410[*OCPBUGS-5410*])
```

Comment:
@lpettyjo could you let us know if this is bug fix or should be moved to enhancements, please.

Reply:
> This one also seems more like an enhancement or change in functionality with an accompanying workaround rather than a bug fix.

@lpettyjo could you let us know if this is bug fix or should be moved to enhancements, please.

Reply:
@JoeAldinger looks like enhancement.